contrib/actorize.py

```diff
@@ -0,0 +1,67 @@
+"""
+Define handlers using decorators
+
+Usage:
+
+@Actorize
+class Hello(Actor):
+
+    @handle(SomeMessageClass)
+    def somemethod(self, msg, sender):
+        pass
+
+
+"""
+
+import inspect
+
+from thespian.actors import Actor
+
+def Actorize(cls):
+
+    def receiveMessage(self, message, sender):
+        klass = message.__class__
+        registry = self._msg_registry
+        if not klass in registry:
+             for k in registry:
+                if isinstance(message, k):
+                    registry[klass] = registry[k]
+                    break
+        method = registry.get(klass, None)
+        if method is not None:
+            method(self, message, sender)
+        else:
+            super(cls, self).receiveMessage(message, sender)
```

Comment:
You may want to update the docstring to describe what happens if there is no registered method in the Actorize'd class for the message received.  If I understand the implementation here correctly, any `receiveMessage` in cls itself will be overwritten by this instance here, but a `receiveMessage` in a baseclass of cls will get an opportunity to handle unrecognized messages?